### DISCUSSION TITLE
Simple list of forms as standard href links

### DISCUSSION THREAD

**Discussion by wolispace** [solspace/craft-freeform#1461] [Feature Requests]
Before Freeform 5, we had a nice simple list of forms as as standard href links.

Great for managing the forms as you could open the links in a new tab and see the full long names of the forms.

Please can we have this as a list option again, or is there already a simple list like this?

Thanks

**Comment by kjmartens** [comment]
Thanks for your feedback @wolispace.

What we will likely consider here would be a list vs cards option for the current "Forms" page in the CP. 🙂

**Reply by wolispace** [reply]
This would be excellent.

Please allow them to be simple a href links so we can right-click and open in a new tab.

It could be as simple as an little icon next to the h2 that is now the current way of opening the form.

Managing a site with about 20 forms, that all need to be opened and reviewed concurrently, is best done with them in multiple tabs not having to go back and forth between them.

Or is there a way of doing this now in Freeform5 as there was in previous versions?

Thanks

**Comment by kjmartens** [comment]
Just thought I'd provide an update on this one... in Freeform 5.6 we added support for a List view of forms. 🙂

**Comment by wolispace** [comment]
Yes.. the list view is great, just wait I find useful. Thanks
